feat: add env variable to skip binary download on npm install (#17627)
electron/electron: docs/tutorial/installation.md

npm install --platform=win32 electron
```

> ## Skip binary download

Reviewer: Can we move this after `## Cache`

> When installing electron for the first time using npm, it automatically downloads the electron binary.

Reviewer: ```suggestion
When installing the `electron` NPM package, it automatically downloads the electron binary.
```

This can sometimes be unnecessary, e.g. in a CI environment, when testing another component.

> To prevent the binary from being downloaded when you install all npm dependencies just set the environment variable `ELECTRON_BINARY_SKIP_DOWNLOAD`.

Reviewer: ```suggestion
To prevent the binary from being downloaded when you install all npm dependencies you can set the environment variable `ELECTRON_BINARY_SKIP_DOWNLOAD`.
```

E.g.:
```sh

> ELECTRON_BINARY_SKIP_DOWNLOAD=1 npm install

Reviewer: Can we make this read a bit clearer by flipping the order

```suggestion
ELECTRON_SKIP_BINARY_DOWNLOAD=1 npm install
```
